config/locales/active_record.he.yml

```diff
@@ -19,6 +19,8 @@ he:
       featured_content: תוכן מערכת
       person: אדם
     errors:
+      messages:
+        required: required
```

Comment:
need translation (this message is shown when mandatory field is missing in ActiveRecord object)